docker/osfamilymap.yaml

```diff
@@ -10,3 +10,9 @@ RedHat:
     key_url: https://download.docker.com/linux/{{ grains['os'] |lower }}/gpg
     version: {{ grains['oscodename'] |lower }}
     file: /etc/yum.repos.d/docker-ce.repo
+Suse:
+  pkg:
+    name: docker
+    version: latest
+  pip:
+    pkgname: python2-pip
```

Comment:
same thing should be python-pip which will in turn give python2-pip or python3-pip based on suse version.

Reply:
Going to retest this.

Reply:
This is silly too. So python-pip will not work on OpenSUSE?
```
          ID: pip_req_pkgs
    Function: pkg.installed
      Result: False
     Comment: An error was encountered while installing package(s): Zypper command failure: Package 'python-pip' not found.
     Started: 09:19:04.574598
    Duration: 1556.153 ms
     Changes:


opensuse-leap-x64:/home/vagrant/opensds-installer/salt # zypper in python-pip
Loading repository data...
Reading installed packages...
'python-pip' not found in package names. Trying capabilities.
'python2-pip' providing 'python-pip' is already installed.
Resolving package dependencies...

Nothing to do.
```

Reply:
no it's because python2-pip already provides python-pip, and it's correct that is already installed..

Reply:
I think the transition of py2/py3 isn't finished, so far as I know, the idea was to have python2-module and python3-module installed together, but installing the python-module would behave for newer "tumbleweed" or "leap somever" depend on python3-module and older versions depend on python2-module...

Reply:
What do you think is best here?  Should I test again without any python-pips installed?  I kinda expected python-pip installation to work.   Is python2-pip correct for both PRs on Suse?

Reply:
@noelmcloughlin just make the package mutable via pillar, and default to python2-pip.
most of suse / sles is still on python2

Reply:
so let's merge this

Reply:
good suggestion.

Reply:
Is that last commit okay - and correct way?